RELEASING.md

```diff
@@ -2,80 +2,16 @@
 
 This describes how to release Dagger:
 
-- [🚙 Engine + 🚗 CLI ⏱ `30mins`](#-engine---cli--30mins)
-- [🐹 Go SDK ⏱ `10mins`](#-go-sdk--10mins)
-- [🐍 Python SDK ⏱ `5mins`](#-python-sdk--5mins)
-- [⬢ TypeScript SDK ⏱ `5mins`](#-typescript-sdk--5mins)
-- [🧪 Elixir SDK ⏱ `5mins`](#-elixir-sdk--5mins)
-- [🐘 PHP SDK ⏱ `5mins`](#-php-sdk--5mins)
-- [☸️ Helm chart ⏱ `2mins`](#%EF%B8%8F-helm-chart--2mins)
+- [📝 Preparation ⏱ `30mins`](#-preparation--30mins)
+- [🚀 Release ⏱ `10mins`](#-release--10mins)
 - [📒 Documentation ⏱ `5mins`](#-documentation--5mins)
 - [🌌 Daggerverse ⏱ `2mins`](#-daggerverse--2mins)
-- [☁️ Dagger Cloud ⏱ `2mins`](#-dagger-cloud--2mins)
-- [🪣 Install scripts ⏱ `2mins`](#-install-scripts--2mins#)
-- [🐙 dagger-for-github ⏱ `10mins`](#-dagger-for-github--10mins#)
-- [🍺 dagger Homebrew ⏱ `2mins`](#-dagger-homebrew--2mins#)
-- [❄️ nix ⏱ `2mins`](#-nix--2mins#)
-- [⚙️ CI ⏱ `2mins`](#-ci--2mins)
-
-This is a high-level diagram of how all the pieces fit together:
-
-```mermaid
-flowchart TB
```

Comment:
So with such a large refactor of the process, this probably needs updating - but a couple things:

- Is this actually helpful? I find mermaid's rendering slightly unhelpful to actually improve understanding.
- This is *very* hard to update - updating the diagram means lots of weights often need rebalancing to get a sane result.
- What's the actual purpose of this diagram? Is it to understand the process? The resulting artifacts?

I think it's worth removing.